content/middlewares.md

```diff
@@ -44,6 +48,36 @@ export class LoggerMiddleware {
 }
 ```
 
+#### Use with Fastify
```

Comment:
I'm wondering if this chapter is a good place for this. IMO we should move this example to the Fastify page and just add a HINT blockquote (or warning blockquote) linking to that page

Reply:
It's indifferent to me, tell me if I should move it to that page 👍

@jmcdo29 what do you think?

Reply:
I've updated this PR by moving the example of using fastify middleware to the fastify dedicated page, leaving just a notice with the link to the page. see commit here: https://github.com/nestjs/docs.nestjs.com/pull/2589/commits/66779cc31210fcff3582e873c37275a0ee5ed768
When you find some time let me know if it's okay how I set it up 🙌